.template.yml

```diff
@@ -0,0 +1,100 @@
+remote_theme: "carpentries-i18n/carp-theme"
```

Comment:
This file is to reset the config for a multilingual English lesson. I've done this manually for the R-gapminder lessons but haven't scripted this yet. Ended up down a rabbit hole with submodules...